website/src/css/_navbar.scss

```diff
@@ -22,3 +22,47 @@
   padding: 5px;
   font-weight: 300;
 }
+/*===========*/
+
+/* Fix navbar logo */
+.navbar__logo {
+  height: 26px;
+  margin: 0 10px 0 8px;
+}
+
+/* Restyle gamburger button */
+
+.navbar {
+  position: relative;
+  height: 70px;
+}
+
+.navbar__toggle {
+  width: 40px;
+  height: 40px;
+  background-color: var(--detox-color-a3);
+  border-radius: 10px;
+  justify-content: center;
+  align-items: center;
+  position: absolute;
+  margin-right: 0;
+  top: 15px;
+  right: 25px;
+}
+
+/* Restyle search button */
+
+@media (max-width: 768px) {
+  button.DocSearch-Button {
+    background-color: var(--detox-color-a3);
+    height: 40px;
+    border-radius: 10px;
+    padding: 0 10px;
+  }
+  .DocSearch-Search-Icon {
+    opacity: 0.6;
+  }
+}
+div.searchBox_node_modules-\@docusaurus-theme-classic-lib-theme-Navbar-Search-styles-module {
```

Comment:
Ugly and unreliable selector. It won't work in the production mode. 

<img width="256" alt="image" src="https://user-images.githubusercontent.com/1962469/218151308-98ae9f41-b8f5-4f38-93ee-99b58be56519.png">

Reply:
Module (scoped) CSS classes are autogenerated. If you are unsure when working on such changes, you can use two commands: `npm run build` and `npm run serve`. The first will build the production-ready static website, and the second one will run a simple HTTP server serving the compiled files.

Reply:
Sorry, didn't know it didn't work in production mode. In general, it is not a problem to bind to the nearest parent element. Although top/right calculations will not be so obvious.